client/extensions/woocommerce/app/store-stats/store-stats-widget-list/index.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { find, findIndex, isEqual } from 'lodash';
+import { translate, moment } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Table from 'woocommerce/components/table';
+import TableRow from 'woocommerce/components/table/table-row';
+import TableItem from 'woocommerce/components/table/table-item';
+import Card from 'components/card';
+import QuerySiteStats from 'components/data/query-site-stats';
+import {
+	isRequestingSiteStatsForQuery,
+	getSiteStatsNormalizedData
+} from 'state/stats/lists/selectors';
+import StatsModulePlaceholder from 'my-sites/stats/stats-module/placeholder';
+import ErrorPanel from 'my-sites/stats/stats-error';
+import Sparkline from '../../../components/sparkline';
+import Delta from '../../../components/delta';
+
+class StoreStatsWidgetList extends Component {
+
+	static propTypes = {
+		data: PropTypes.object.isRequired, // TODO refactor to array :(
+		emptyMessage: PropTypes.string,
+		selectedDate: PropTypes.string.isRequired,
+		siteId: PropTypes.number,
+		statType: PropTypes.string,
+		query: PropTypes.object.isRequired,
+		widgets: PropTypes.array.isRequired,
+	};
+
+	state = {
+		loaded: false
+	};
+
+	componentWillReceiveProps( nextProps ) {
+		if ( ! nextProps.requesting && this.props.requesting ) {
+			this.setState( { loaded: true } );
+		}
+		if ( ! isEqual( nextProps.query, this.props.query ) && this.state.loaded ) {
+			this.setState( { loaded: false } );
+		}
+	}
+
+	getEndPeriod = ( date ) => {
+		const { unit } = this.props.query;
+		return ( unit === 'week' )
+			? moment( date ).endOf( 'isoWeek' ).format( 'YYYY-MM-DD' )
+			: moment( date ).endOf( unit ).format( 'YYYY-MM-DD' );
+	};
+
+	getDeltasBySelectedPeriod = () => {
+		return find( this.props.data.deltas, ( item ) => item.period === this.props.selectedDate );
+	};
+
+	getDeltaByStat = ( stat ) => {
+		return this.getDeltasBySelectedPeriod()[ stat ];
+	};
+
+	getSelectedIndex = ( data ) => {
+		return findIndex( data, d => d.period === this.props.selectedDate );
+	};
+
+	render() {
+		const { selectedDate, siteId, statType, query, data, emptyMessage, widgets } = this.props;
+		const selectedIndex = this.getSelectedIndex( data.data );
+		const { loaded } = this.state;
+		const isLoading = ! loaded && ! ( data && data.length );
+		const hasEmptyData = loaded && data && data.length === 0;
+		const values = [
+			{
+				key: 'title',
+				label: translate( 'Stat' )
+			},
+			{
+				key: 'value',
+				label: translate( 'Value' )
+			},
+			{
+				key: 'sparkline',
+				label: translate( 'Trend' )
+			},
+			{
+				key: 'delta',
+				label: translate( 'Since ___' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **55** times:
`translate( 'Since' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).